server/reload.go

```diff
@@ -1143,28 +1143,23 @@ func (s *Server) reloadClusterPermissions(oldPerms *RoutePermissions) {
 	// Regenerate route INFO
 	s.generateRouteInfoJSON()
 	infoJSON = s.routeInfoJSON
-	s.mu.Unlock()
```

Comment:
Wonder if you could capture account here (sgacc := s.gacc), instead of moving the unlock down there?
But, doing that way, it is possible that if there is a config reload and s.gacc is captured before, then we would remove from old, not from the new one, so maybe "safer" this way.

Reply:
That was my original train of thought. 
I guess, since this is the function reloadClusterPermissions it'd probably be safe to capture acc.

Reply:
Right, since this is invoked during the config reload, I would then definitively minimize locking changes by just capturing s.gacc.